server/configuration/accounts_meld.js

```diff
@@ -0,0 +1,47 @@
+const orig_updateOrCreateUserFromExternalService = Accounts.updateOrCreateUserFromExternalService;
+
+Accounts.updateOrCreateUserFromExternalService = function(serviceName, serviceData = {} /*, options*/) {
+	const services = [
+		'facebook',
+		'github',
+		'gitlab',
+		'google',
+		'meteor-developer',
+		'linkedin',
+		'twitter',
+		'sandstorm'
+	];
+
+	if (services.includes(serviceName) === false && serviceData._OAuthCustom !== true) {
+		return;
+	}
+
+	if (serviceName === 'meteor-developer') {
+		if (Array.isArray(serviceData.emails)) {
+			serviceData.email = serviceData.emails.sort(a => a.primary !== true).filter(item => item.verified === true)[0];
```

Comment:
Won't it throw an error because you're trying to access property [0]?

`serviceData.email = serviceData.emails.sort(a => a.primary !== true).filter(item => item.verified === true)[0];`